sherpa/plot/tests/test_plot.py

```diff
@@ -115,9 +115,9 @@ def test_splitplot(self):
         # sp.addplot(rp)
 
 
+@requires_data
 class test_contour(SherpaTestCase):
```

Comment:
Good idea. I started looking into ways to test the output of the plotting backends, but that's not a priority unless we make it such.